examples/camera/camera_orbit.rs

```diff
@@ -0,0 +1,163 @@
+            ..default()
+        },
+    ));
+}
+
+fn instructions(mut commands: Commands) {
+    commands
+        .spawn((
+            Name::new("Instructions"),
+            NodeBundle {
+                style: Style {
+                    align_items: AlignItems::Start,
+                    flex_direction: FlexDirection::Column,
+                    justify_content: JustifyContent::Start,
+                    width: Val::Percent(100.),
+                    ..default()
+                },
+                ..default()
+            },
+        ))
+        .with_children(|parent| {
+            parent.spawn(TextBundle::from_section(
+                "W or S: pitch",
+                TextStyle::default(),
+            ));
+            parent.spawn(TextBundle::from_section(
+                "A or D: yaw",
+                TextStyle::default(),
+            ));
+            parent.spawn(TextBundle::from_section(
+                "Q or E: roll",
+                TextStyle::default(),
+            ));
+        });
+}
+
+fn orbit(
+    mut camera: Query<&mut Transform, With<Camera>>,
+    camera_settings: Res<CameraSettings>,
+    keyboard_input: Res<ButtonInput<KeyCode>>,
+    time: Res<Time>,
+) {
+    let mut transform = camera.single_mut();
+
+    let mut delta_pitch = 0.0;
+    let mut delta_roll = 0.0;
+    let mut delta_yaw = 0.0;
+
+    if keyboard_input.pressed(KeyCode::KeyW) {
+        delta_pitch += camera_settings.orbit_speed;
+    }
+    if keyboard_input.pressed(KeyCode::KeyS) {
+        delta_pitch -= camera_settings.orbit_speed;
+    }
+
+    if keyboard_input.pressed(KeyCode::KeyQ) {
+        delta_roll -= camera_settings.orbit_speed;
+    }
+    if keyboard_input.pressed(KeyCode::KeyE) {
+        delta_roll += camera_settings.orbit_speed;
+    }
+
+    if keyboard_input.pressed(KeyCode::KeyA) {
+        delta_yaw -= camera_settings.orbit_speed;
+    }
+    if keyboard_input.pressed(KeyCode::KeyD) {
+        delta_yaw += camera_settings.orbit_speed;
+    }
+
+    // Incorporating the delta time between calls prevents this from being framerate-bound.
+    delta_pitch *= time.delta_seconds();
+    delta_roll *= time.delta_seconds();
+    delta_yaw *= time.delta_seconds();
+
+    // Obtain the existing pitch, yaw, and roll values from the transform.
+    let (yaw, pitch, roll) = transform.rotation.to_euler(EulerRot::YXZ);
+
+    // Establish the new yaw and pitch, preventing the pitch value from exceeding our limits.
+    let pitch = (pitch + delta_pitch).clamp(
+        camera_settings.pitch_range.start,
+        camera_settings.pitch_range.end,
+    );
+    let roll = roll + delta_roll;
+    let yaw = yaw + delta_yaw;
+    transform.rotation = Quat::from_euler(EulerRot::YXZ, yaw, pitch, roll);
+
+    // Adjust the translation to maintain the correct orientation toward the orbit target.
+    transform.translation = Vec3::ZERO - transform.forward() * camera_settings.orbit_distance;
```

Comment:
```suggestion
    // Adjust the translation to maintain the correct orientation toward the orbit target.
    let target = Vec3::ZERO;
    transform.translation = target - transform.forward() * camera_settings.orbit_distance;
```
That should make it a bit more obvious how to customize this code